src/main/java/edu/kis/powp/jobs2d/command/ComplexCommandFactory.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+public class ComplexCommandFactory {
+    public static void drawRectangle(int x, int y, int off1, int off2) {
```

Comment:
If the class is a factory you should return commands instead of executing (it would provide greater reuseability)